__tests__/integration/mint-tokens.test.js

```diff
@@ -284,7 +357,7 @@ describe('mint token', () => {
     expect(transaction.success).toBe(true);
     await TestUtils.waitForTxReceived(wallet1.walletId, response2.body.hash);
 
-    const addr17 = await wallet1.getAddressInfo(17, tokenA.uid);
+    const addr17 = await wallet1.getAddressInfo(20, tokenA.uid);
```

Comment:
Should we change this variable name to `addr20` to prevent confusion?

Reply:
Done